app/admin/listings/[id]/page.tsx

```
@@ -0,0 +1,176 @@
+            <CardHeader>
+              <CardTitle>Media Gallery</CardTitle>
+            </CardHeader>
+            <CardContent className="grid grid-cols-2 md:grid-cols-3 gap-4">
+              {listing.images?.map((img, index) => (
+                <div
+                  key={index}
+                  className="aspect-square bg-muted rounded-lg overflow-hidden"
+                >
+                  <img
+                    src={img}
+                    alt={`Listing image ${index + 1}`}
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)